src/OrchardCore.Themes/TheComingSoonTheme/wwwroot/package-lock.json

```diff
@@ -1,7203 +1,8 @@
 {
     "name": "startbootstrap-coming-soon",
     "version": "5.0.9",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```

Comment:
This should stay to version 2. You need to upgrade your npm version to latest or node.js.